package-lock.json

```diff
@@ -94,6 +94,15 @@
         }
       }
     },
+    "JSONStream": {
```

Comment:
do we need these changes?

Reply:
No, any changes to the `package-lock.json` are based on local package preferences, and aren't required for this PR, I'll remove that.